azure/packages/azure-client/package.json

```diff
@@ -44,31 +44,31 @@
 	"dependencies": {
 		"@fluidframework/common-definitions": "^0.20.1",
 		"@fluidframework/common-utils": "^1.1.1",
-		"@fluidframework/container-definitions": "workspace:~",
-		"@fluidframework/container-loader": "workspace:~",
-		"@fluidframework/core-interfaces": "workspace:~",
-		"@fluidframework/driver-definitions": "workspace:~",
-		"@fluidframework/driver-utils": "workspace:~",
-		"@fluidframework/fluid-static": "workspace:~",
-		"@fluidframework/map": "workspace:~",
+		"@fluidframework/container-definitions": ">=2.0.0-internal.4.4.4 <2.0.0-internal.4.5.0",
```

Comment:
Are all those dependency changes from "workspace:~" to ">=2.0.0-internal.4.4.4 <2.0.0-internal.4.5.0" expected?

Reply:
It doesn't look like they should be there. I'm closing this PR for a new one, that uses the original flub script (not the fancy tip script) in the [instructions](https://eng.ms/docs/experiences-devices/opg/office-shared/fluid-framework/fluid-framework-internal/fluid-framework/docs/on-call/release/release-patch?tabs=release-groups), and produces a more expected diff: https://github.com/microsoft/FluidFramework/pull/16599.


<img width="975" alt="Screenshot 2023-07-27 at 12 57 38 PM" src="https://github.com/microsoft/FluidFramework/assets/6777404/55fd0d32-1a44-4664-bd38-eeccc579cb77">